packages/internal-tooling/babel/index.js

```diff
@@ -7,11 +7,11 @@ module.exports = {
   plugins: [
     '@babel/plugin-proposal-class-properties',
     '@babel/plugin-syntax-dynamic-import',
-    '@babel/plugin-transform-runtime',
+    ['@babel/plugin-transform-runtime', { corejs: 2 }],
   ],
   env: {
-    development: {
-      plugins: ['react-hot-loader/babel'],
```

Comment:
Moved to Webpack's config directly, should fix issues with perf tests
FYI: @miroslavstastny